reindexer/get_reindex_status.py

```diff
@@ -242,6 +275,21 @@ def main(reindex_date):
 
     print("")
 
+    print("*** Queue stats ***")
+    dlq_stats = get_dlq_stats(session_read_only, reindex_date=reindex_date)
+    non_empty_dlqs = {q_url: size for q_url, size in dlq_stats.items() if size > 0}
+    if non_empty_dlqs:
+        print("The following DLQs have failed messages:")
+        rows = [
+            [q_url.split("/")[-1].replace(f"catalogue-{reindex_date}_", ""), size]
+            for q_url, size in sorted(non_empty_dlqs.items())
+        ]
+        print(tabulate.tabulate(rows, tablefmt="plain", colalign=("left", "right")))
+    else:
+        print("There are no messages on DLQs")
+
+    print("")
+
```

Comment:
This change is to highlight any DLQs in this pipeline that have failing messages:

<img width="735" alt="Screenshot 2021-09-24 at 10 15 15" src="https://user-images.githubusercontent.com/301220/134650364-adf94572-369a-464f-b6fc-bf8ed93b8b28.png">